recipes/pretty_html_table/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "pretty_html_table" %}
+{% set version = "0.9.dev0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "63b3ce16ab1abdbc4888094ba792908666122eafe3fd98ed127efbb4dd3ac0f9"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pandas
+    - pip
+    - python
+  run:
+    - pandas
+    - python
+
+test:
+  imports:
+    - pretty_html_table
+
+about:
+  home: "https://github.com/sbi-rviot/ph_table"
+  license: "MIT"
+  license_family: "MIT"
+  license_file: "LICENSE.txt"
```

Comment:
You're going to need to manually package the license file:

https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually

also, please add a comment in the line above `license_file` including the link to the pull request below. This will be a reminder to check to see if the license file is in the source distribution of future releases of the package so the manually added copy can be removed. 

https://github.com/sbi-rviot/ph_table/pull/5